### DISCUSSION TITLE
Winpilot 2024.6.3

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/Bloatynosy#217] [Announcements]
# Changelog
- Winpilot has been tested on various Dev and Release preview builds and fine-tuned in detail. **Support for 24H2 is officially available as of today.**

![vmconnect_NoCSYBuAys](https://github.com/builtbybel/Winpilot/assets/57478606/320a601a-aa2c-43bf-9554-48cb9022056f)

Winpilot should no longer (although never say never) be classified as a "potentially unwanted app" by Microsoft Defender or Microsoft Defender SmartScreen.  It's like I can feel that the app is now welcome at Microsoft and no longer gets flagged 😸 
The app is completely legitimate and does not use any aggressive mechanisms to "break things". Instead, it simply toggles registry switches and uses PowerShell to uninstall apps (just like you do when uninstalling apps from the Start menu). This excludes external scripts like the currently experimental tiny11builder.

## Warning: Scam Alert
**Please be cautious about where you download the app. I've seen several modified versions of Winpilot as well as Bloatynosy, which I wouldn't recommend using. Also, avoid the site bloatynosy.com. It promotes Bloatynosy as the official source, but you'll end up downloading an old version of Winpilot. Unfortunately, I get emails from people who have downloaded from there, asking what’s going on (I haven't examined the binaries yet). It seems forums and Reddit are unfortunately referring to that domain.**

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/Winpilot/releases/tag/2024.6.3'>Winpilot 2024.6.3</a>.</em>

**Comment by c0nape** [comment]
Will the app have Russian language?
